django_redis/client/default.py

```diff
@@ -299,6 +299,9 @@ def expire(
         version: Optional[int] = None,
         client: Optional[Redis] = None,
     ) -> bool:
+        if (timeout is DEFAULT_TIMEOUT) or (timeout is None):
```

Comment:
I would let people set None as timeout and get the exception from the client so that they notice they made a mistake instead of getting a seemingly random expiry time

Reply:
this one is missing

Reply:
Sorry this was my bad. Fixed!